apps/emqx/src/emqx_schema.erl

```diff
@@ -3385,3 +3412,18 @@ ensure_default_listener(Map, ListenerType) ->
 
 cert_file(_File, client) -> undefined;
 cert_file(File, server) -> iolist_to_binary(filename:join(["${EMQX_ETC_DIR}", "certs", File])).
+
+mqtt_converter(undefined, _Opts) ->
+    undefined;
+mqtt_converter(#{} = Mqtt, _Opts) ->
+    Multiplier = maps:get(<<"keepalive_multiplier">>, Mqtt, ?DEFAULT_MULTIPLIER),
+    Backoff = maps:get(<<"keepalive_backoff">>, Mqtt, ?DEFAULT_BACKOFF),
+    case Multiplier == Backoff * 2 of
+        true ->
+            Mqtt;
+        false when Backoff == ?DEFAULT_BACKOFF -> Mqtt;
+        false when Multiplier == ?DEFAULT_MULTIPLIER ->
+            Mqtt#{<<"keepalive_multiplier">> => Backoff * 2};
+        false ->
+            Mqtt
+    end.
```

Comment:
We should cover when use modify backoff and multiplier at the same time.
if modify both, we should use multiplier instead of backoff * 2.
such as #{backoff => 1, multiplier => 5} 's result should #{multiplier => 5} not 1*2.